artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationValidationUtils.java

```diff
@@ -0,0 +1,80 @@
+package org.apache.activemq.artemis.ra.inflow;
```

Comment:
RAT failure, requires license headers

Reply:
Same, my bad. I knew I needed to double check the license headers and forgot to do it. I'll update the PR accordingly.